tools/ci/azure/update_hosts.yml

```diff
@@ -5,6 +5,6 @@ steps:
 - powershell: |
     $hostFile = "$env:systemroot\System32\drivers\etc\hosts"
     Copy-Item -Path $hostFile -Destination "$hostFile.back" -Force
-    python wpt make-hosts-file | Out-File $env:systemroot\System32\drivers\etc\hosts -Encoding ascii -Append
+    python wpt --py2 make-hosts-file | Out-File $env:systemroot\System32\drivers\etc\hosts -Encoding ascii -Append
```

Comment:
What's the reason for using py2 here?

Reply:
Whoops, forgot to update the PR description when I updated the commit description. Updated now, and borrowing from it:

> Some Azure Pipelines helper scripts are used for both py2 and py3 tasks.
> As a simple way to keep them working, --py2 is used for them as it is
> always available.

We could (if we felt fancy) introduce a variable here and have py2 tasks set it to true, but I just wanted to keep things simple since we're going py3-only so soon anyway.